kuma/health/tests/test_views.py

```diff
@@ -12,35 +12,43 @@
 
 
 @pytest.mark.parametrize('http_method', ['put', 'post', 'delete', 'options'])
-@pytest.mark.parametrize('endpoint', ['health.liveness', 'health.readiness'])
+@pytest.mark.parametrize('endpoint', ['liveness', 'readiness', 'status'])
 def test_disallowed_methods(client, http_method, endpoint):
     """Alternate HTTP methods are not allowed."""
-    url = reverse(endpoint)
+    url = reverse('health.{}'.format(endpoint))
     response = getattr(client, http_method)(url)
     assert response.status_code == 405
+    assert 'Cache-Control' in response
+    assert 'max-age=0' in response['Cache-Control']
```

Comment:
Nit: If the ``Cache-Control`` header is missing, then checking for ``max-age=0`` will also fail, so checking for ``Cache-Control`` is redundant.